pkg/cli/mirror/catalog_images.go

```diff
@@ -154,18 +159,20 @@ func (o *MirrorOptions) rebuildCatalogs(ctx context.Context, dstDir string, file
 			}
 			layers = append(layers, add)
 
-			srcImage = operator.OPMImage
+			opmImage, err := reference.Parse(operator.OPMImage)
+			if err != nil {
+				return refs, fmt.Errorf("error parsing image %q: %v", operator.OPMImage, err)
+			}
+			// Update registry so the existing OPM image can be pulled.
+			// QUESTION(jpower): is assuming an image is present in a repo with the same name valid?
```

Comment:
This could be configured in the future, but it's totally fine to make this assumption for now since the opm image is added automatically to the imageset during archive creation.